### DISCUSSION TITLE
When updaing models, respect original file's folder

### DISCUSSION THREAD

**Discussion by Sir-Digsbey** [BlafKing/sd-civitai-browser-plus#169] [Ideas]
I sort my models and LORAs in sub-folders.

When using the updater, it downloads the new models and LORAs to the root of the respective folder (e.g. `models/Lora`, `models/Stable-diffusion` etc.) instead of the same folder as the model it's updating (e.g. `models/Lora/Clothes`, 'models/Stable-Diffusion/Realistic` etc.).

It would be nice if it downloaded it to the same folder as the one it's updating.

**Comment by BlafKing** [comment]
Thanks for notifying me about it!

This should only happen if you're using multi selected downloading, which now checks if the model is already installed, and if so, will use the same folder as any previous versions if no other sub folder was provided!

This has been implemented in v3.3.0! :)
